### PR TITLE
Add Korean translation

### PR DESCRIPTION
**Pull request by VenusGirl**
Thank you for your contribution to the Sandboxie repository.

In order to reduce the risk of accidental merges, it's highly recommended for beginners to open a new Pull Request in draft state by clicking the button "Create Draft Pull Request", instead of using the default "Create Pull Request" option. 

In addition, you can convert an existing pull request to a draft by clicking the "Convert to draft" link in the right sidebar under "Reviewers".

All new translators are encouraged to look at the "Localization notes and tips" before creating a pull request: https://git.io/J9G19

### CODE REVIEW

**Review by isaak654 — CHANGES_REQUESTED** [review]
That's promising! 

Other dependencies to translate / add:

- [ ] 1. <a href="https://github.com/sandboxie-plus/Sandboxie/blob/master/Installer/Languages.iss">Here</a> you can find the location of the `Sandboxie Plus` installer languages file where you need to translate the english lines in the [CustomMessages] section

- [ ] 2. <a href="https://github.com/sandboxie-plus/Sandboxie/blob/56396e62f0a634b684d87cdeeb299253d6a77b9c/Installer/Sandboxie-Plus.iss#L206-L224">Here</a> you can find the location of the `Sandboxie Plus` installer script where you need to add the language value, for example: `'korean': Result := 'ko';`

- [ ] 3. Upload the [Korean unofficial language for InnoSetup](https://github.com/jrsoftware/issrc/blob/main/Files/Languages/Unofficial/Korean.isl) in our [custom ISL folder](https://github.com/sandboxie-plus/Sandboxie/tree/56396e62f0a634b684d87cdeeb299253d6a77b9c/Installer/isl)

- [ ] 4. Add `Name: "korean"; MessagesFile: "isl\Korean.isl"` in the [Languages] section of [Languages.iss](https://github.com/sandboxie-plus/Sandboxie/blob/56396e62f0a634b684d87cdeeb299253d6a77b9c/Installer/Languages.iss)

File: SandboxiePlus/SandMan/sandman_ko.ts [hunk 1]
```diff
@@ -0,0 +1,5377 @@
...
+        <translation type="unfinished">제거된 템플릿을 정리했습니다...</translation>
+    </message>
+    <message>
+        <location filename="SandMan.cpp" line="1499"/>
+        <source>Sandboxie-Plus was started in portable mode, do you want to put the Sandbox folder into its parent directory?
+Yes will choose: %1
+No will choose: %2</source>
+        <translation type="unfinished">        <translation type="unfinished">Sandboxie-Plus가 휴대용 모드로 시작되었습니다. Sandbox 폴더를 상위 디렉터리에 넣으시겠습니까?
+예 선택: %1
+아니요 선택: %2</translation></translation>
```
Comment (isaak654) [inline SandboxiePlus/SandMan/sandman_ko.ts:1686]:
```suggestion
        <translation type="unfinished">Sandboxie-Plus가 휴대용 모드로 시작되었습니다. Sandbox 폴더를 상위 디렉터리에 넣으시겠습니까?
예 선택: %1
아니요 선택: %2</translation>
```

**Review by isaak654 — CHANGES_REQUESTED** [review]

**Review by isaak654 — CHANGES_REQUESTED** [review]
As a rule of thumb, you shouldn't ever translate the .ts files by hand, in order to avoid formatting issues (the use of an official translation tool like `Qt Linguist` is highly recommended).
 
Just in case, please see also the other recommendations in the Localization notes and tips: https://git.io/J9G19

File: SandboxiePlus/SandMan/sandman_ko.ts [hunk 2]
```diff
@@ -0,0 +1,5377 @@
...
+    <message>
+        <location filename="Windows/SettingsWindow.cpp" line="858"/>
+        <location filename="Windows/SettingsWindow.cpp" line="972"/>
+        <source>Select Directory</source>
+        <translation type="unfinished">디렉터리 선택</translation>
+    </message>
+    <message>
+        <location filename="Windows/SettingsWindow.cpp" line="881"/>
+        <source>&lt;a href=&quot;check&quot;&gt;Check Now&lt;/a&gt;</source>
+        <translation type="unfinished">&lt;a href=&quot;check&quot;&gt;지금 확인&lt;/a&gt;<</translation>
```
Comment (isaak654) [inline SandboxiePlus/SandMan/sandman_ko.ts:3099]:
```suggestion
        <translation type="unfinished">&lt;a href=&quot;check&quot;&gt;지금 확인&lt;/a&gt;</translation>
```

**Review by isaak654 — CHANGES_REQUESTED** [review]
At this rate we will not make it, please use the official Linguist tool described in the previous guide to check all errors at once instead to wait the reviews one by one (there are more than 5000 lines in sandman_ko.ts).
Once loaded the TS file, the tool will indicate you the entries to fix. It will also help you to prevent any type of formatting issue in future.

EDIT: Explained better in https://github.com/sandboxie-plus/Sandboxie/pull/2133#pullrequestreview-1072117923

**Review by isaak654 — CHANGES_REQUESTED** [review]
* We're almost there, now you need to open the TS file. You will see this:
linguist_pic

- [x] 1. the esclamation marks in red mean that some element of punctuation is missing. (fixed with https://github.com/sandboxie-plus/Sandboxie/pull/2133/commits/2faa51bdf4de72590e172649bcf02e7484247d3f)
- [ ] 2. the questions marks in yellow mean that you need to mark the translated entries as finished: (To be done)
entry_marked_as_finished
Of course, you're free to leave the entries of which you have doubts as unfinished, but please do not keep <ins>all of them as unfinished</ins>.

How_to_mark_finished_translations_in_Linguist

**Comment by isaak654** [comment]
👉 https://github.com/sandboxie-plus/Sandboxie/pull/2133#pullrequestreview-1072117923 (updated)

**Comment by isaak654** [comment]
If you have doubts about the remaining task in https://github.com/sandboxie-plus/Sandboxie/pull/2133#pullrequestreview-1072117923, I am available for any clarification.

**Comment by VenusGirl** [comment]
#2143 is a correction of all the missing translations, incorrect translations, and recently changed digit position changes in #2133.

I reviewed it with linguist.exe, and all of them are marked with a yellow question mark.

**Comment by VenusGirl** [comment]
샌드 번역

**Comment by isaak654** [comment]
1. The sandman_ko.ts file in #2143 was identical to the one in this PR, I verified it with the comparison tool WinMerge. But let's say it wasn't, it is better to work on it in one place here.
2. I just uploaded a video at the bottom of [this comment](https://github.com/sandboxie-plus/Sandboxie/pull/2133#pullrequestreview-1072117923) that explains how to mark unfinished entries as finished. I hope it's clear now.

**Comment by VenusGirl** [comment]
샌드박스

**Review by isaak654 — APPROVED** [review]
Ready to be merged, just waiting the CI to be fixed:
CI_fail

@DavidXanatos Make sure to not incur in [#1556](https://cutt.ly/FXgEkAC) after this PR will be merged.

**Comment by VenusGirl** [comment]
Thank you for your help and thank you for your hard work.

**Comment by isaak654** [comment]
You're welcome. 

If you are looking for other ways to contribute:

1. you may also want to adjust [Text-Korean-1042.txt](https://github.com/sandboxie-plus/Sandboxie/blob/master/Sandboxie/msgs/Text-Korean-1042.txt) that is still updated on both Classic & Plus for compatibility reasons.
In order to facilitate the task, I restored a reporting script that periodically lists the new entries that should be added, translated or commented out: [Report-Korean.txt](https://github.com/sandboxie-plus/Sandboxie/blob/master/Sandboxie/msgs/report/Report-Korean.txt)

2. Unfortunately I'm not aware of a clear method to autocommit the new additions from [Report-Korean.txt](https://github.com/sandboxie-plus/Sandboxie/blob/master/Sandboxie/msgs/report/Report-Korean.txt) to [Text-Korean-1042.txt](https://github.com/sandboxie-plus/Sandboxie/blob/master/Sandboxie/msgs/Text-Korean-1042.txt). Just in case, here are the interested scripts: 
- [lupdate.yml](https://github.com/sandboxie-plus/Sandboxie/blob/master/.github/workflows/lupdate.yml) 
- [RunReport.bat](https://github.com/sandboxie-plus/Sandboxie/blob/master/Sandboxie/msgs/RunReport.bat) 
- [Parse.vcxproj](https://github.com/sandboxie-plus/Sandboxie/blob/master/Sandboxie/msgs/Parse.vcxproj) 
- [parse.c](https://github.com/sandboxie-plus/Sandboxie/blob/master/Sandboxie/msgs/parse.c)

**Comment by isaak654** [comment]
You're welcome. 

If you are looking for other ways to contribute:

1. you may want to update [Text-Korean-1042.txt](https://github.com/sandboxie-plus/Sandboxie/blob/master/Sandboxie/msgs/Text-Korean-1042.txt) that is meant to contain both Classic & Plus entries for compatibility reasons.
In order to facilitate the task, I restored a reporting script that periodically lists the new entries that should be added, translated or commented out: [Report-Korean.txt](https://github.com/sandboxie-plus/Sandboxie/blob/master/Sandboxie/msgs/report/Report-Korean.txt)

2. unfortunately I'm not aware of a clear method to autocommit/sync the new additions from [Report-Korean.txt](https://github.com/sandboxie-plus/Sandboxie/blob/master/Sandboxie/msgs/report/Report-Korean.txt) to [Text-Korean-1042.txt](https://github.com/sandboxie-plus/Sandboxie/blob/master/Sandboxie/msgs/Text-Korean-1042.txt). Just in case, here are the interested scripts: 
- [lupdate.yml](https://github.com/sandboxie-plus/Sandboxie/blob/master/.github/workflows/lupdate.yml) 
- [RunReport.bat](https://github.com/sandboxie-plus/Sandboxie/blob/master/Sandboxie/msgs/RunReport.bat) 
- [Parse.vcxproj](https://github.com/sandboxie-plus/Sandboxie/blob/master/Sandboxie/msgs/Parse.vcxproj) 
- [parse.c](https://github.com/sandboxie-plus/Sandboxie/blob/master/Sandboxie/msgs/parse.c)

**Pull request merged**